pkg/services/annotations/annotationsimpl/xorm_store.go

```diff
@@ -377,41 +385,30 @@ func (r *xormRepositoryImpl) Get(ctx context.Context, query annotations.ItemQuer
 	return items, err
 }
 
-func (r *xormRepositoryImpl) getAccessControlFilter(user identity.Requester, accessResources *accesscontrol.AccessResources) (string, error) {
+func (r *xormRepositoryImpl) getAccessControlFilter(user identity.Requester, accessResources *accesscontrol.AccessResources) (string, []any, error) {
 	if accessResources.SkipAccessControlFilter {
-		return "", nil
+		return "", nil, nil
```

Comment:
nit: looks like we never actually return an error - could just remove it from the return parameters.